WolvenKit.App/Controllers/RED4Controller.cs

```diff
@@ -269,15 +267,20 @@ private static bool IsCDPRScript(string fileName)
         };
     }
 
-    private static IEnumerable<string> GetScriptFiles(Cp77Project cp77Proj) 
-        => Directory.EnumerateFiles(cp77Proj.ResourceScriptsDirectory, "*.*", SearchOption.AllDirectories).Where(name => IsCDPRScript(name));
+    private static List<string> GetScriptFiles(Cp77Project cp77Proj)
```

Comment:
why the change to List?

Reply:
ConcurrentModification warnings when accessing

Reply:
ah right, you made it async 👍